src/API/Site/Controllers/BaseReportsController.php

```diff
@@ -39,7 +39,7 @@ public function __construct( ContainerInterface $container ) {
 	 * @return array
 	 */
 	public function get_collection_params(): array {
-		$params = [
```

Comment:
:question: 
Shouldn't we call the parent here as well to get `context, page, per_page, and search` from there?
![image](https://user-images.githubusercontent.com/17435/209560201-d70f93cc-f2da-4a82-a426-12bb8f1538ec.png)

Reply:
I don't think we really can because while we try to reuse as much as possible from the parent controller, we get our data from the Google API instead of from a local DB. Which means some of those query parameters are different.

- context: we redefine this one, although we don't really use context much internally
- page: we can't reuse this because the Google API expects a next page token so we can't query a specific page
- per_page: we redefine this since the Google API sets some different limitations on how many records you can retrieve per request
- search: for the parent controller this is for searching for posts that match a particular search string (again this isn't really appropriate for our controllers)